docs/examples/helpbutton/withZIndex.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+import { type Node } from 'react';
+import { HelpButton, Text, Flex, FixedZIndex } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex height="100%" justifyContent="center" alignItems="center" gap={1}>
+      <Text>This is Gestalt</Text>
+      <HelpButton
+        text="Is Pinterest's design system"
+        accessibilityPopoverLabel="Gestalt meaning description"
+        zIndex={new FixedZIndex(100)}
```

Comment:
Is this zIndex necessary? We don't want to encourage unneeded zIndex usage

Reply:
This example is to show how to use the `zIndex` prop, so I added as a `HelpButton` with `zIndex` usage.
Am I misunderstanding something? 
If your question is about the number `100` is completely random. Can you suggest me a better one?
And if you asking about the usage of `new FixedZIndex` I only keep the approach of `IconButton` since the `IconButton` was the reference/suggestion to use to implement this one. to more infos please read: 
![Screenshot 2023-02-07 at 11 50 33](https://user-images.githubusercontent.com/16820917/217278463-075125e0-c8ee-45eb-a058-ea2748c97bb7.png)

Reply:
🤦 omg this is the z-index example, of course we're using a z-index here. my bad, ignore me 😂